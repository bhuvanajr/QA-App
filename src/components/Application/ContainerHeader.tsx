import React from 'react';

export const ContainerHeader = () => {
  return (
    <div className="head-wrapper">
      <div className="branding">
        <a href="/">
          <img src="/assets/qa_logo.png" alt="Brand logo" className="brand-image" />
        </a>
        QA Tool
      </div>
    </div>
  );
};
