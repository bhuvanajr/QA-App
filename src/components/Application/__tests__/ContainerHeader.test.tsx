import { render } from '@testing-library/react';
import React from 'react';
import { ContainerHeader } from '../ContainerHeader';

test('should render ContainerHeader correctly', () => {
  const { container } = render(<ContainerHeader />);
  expect(container).toMatchSnapshot();
});
