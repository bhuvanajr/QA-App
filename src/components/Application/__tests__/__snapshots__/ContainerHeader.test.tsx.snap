// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ContainerHeader correctly 1`] = `
<div>
  <div
    class="head-wrapper"
  >
    <div
      class="branding"
    >
      <a
        href="/"
      >
        <img
          alt="Brand logo"
          class="brand-image"
          src="/assets/qa_logo.png"
        />
      </a>
      QA Tool
    </div>
  </div>
</div>
`;
